client/extensions/woocommerce/state/action-types.js

```diff
@@ -38,6 +38,8 @@ export const WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST = 'WOOCOMMERCE_SETTINGS_PRODU
 export const WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST_SUCCESS = 'WOOCOMMERCE_SETTINGS_PRODUCTS_REQUEST_SUCCESS';
 export const WOOCOMMERCE_SETUP_CHOICES_REQUEST = 'WOOCOMMERCE_SETUP_CHOICES_REQUEST';
 export const WOOCOMMERCE_SETUP_CHOICES_REQUEST_SUCCESS = 'WOOCOMMERCE_SETUP_CHOICES_REQUEST_SUCCESS';
+export const WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST = 'WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST';
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/004863a3c5f48ce79a2d0ea93ba459112536c873